<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ONIViewDialog</class>
 <widget class="QDialog" name="ONIViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1310</width>
    <height>672</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ONIView</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>1281</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSlider" name="horizontalSlider_playControl">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_pause">
              <property name="text">
               <string>&amp;Pause</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_fullscreenDepth">
              <property name="text">
               <string>FullscreenDepth</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_speed">
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>62</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>135</width>
              <height>62</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QGroupBox{
	border-width:2px;
	border-style:solid;
	border-color:white;
	margin-top:0.5ex;
}
QGroupBox:title{
	subcontrol-origin:margin;
	subcontrol-position:top left;
	left:10px;
	margin-left:0px;
	padding:0 1px;
}</string>
            </property>
            <property name="title">
             <string>Speed</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QComboBox" name="comboBox_playSpeed">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_frameNoShow">
                <property name="minimumSize">
                 <size>
                  <width>67</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>67</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 127);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_totalFrameNumShow">
                <property name="minimumSize">
                 <size>
                  <width>67</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>67</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 127);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_fullscreenColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>FullscreenColor</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pushButton_openOni">
          <property name="text">
           <string>Open&amp;File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_openCam">
          <property name="text">
           <string>Open&amp;Cam</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_quit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Qiut</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_ONIShowColor">
   <property name="geometry">
    <rect>
     <x>656</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Preview area</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ONIShowDepth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Preview area(depth)</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>clicked()</signal>
   <receiver>ONIViewDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1072</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>1001</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
